src/stores/plan.ts

```diff
@@ -1,6 +1,6 @@
 import { derived, writable, type Readable, type Writable } from 'svelte/store';
 import type { ActivityType } from '../types/activity';
-import type { Plan, PlanMergeRequest, PlanMergeRequestSchema } from '../types/plan';
+import { type Plan, type PlanMergeRequest, type PlanMergeRequestSchema, type PlanMetadata } from '../types/plan';
```

Comment:
small nitpick...did this get autoformatted?

Reply:
Yeah autoformatted - maybe this file wasn't reformatted after the prettier updates for some reason? Ran autoformatter again and it continues to do this.

Reply:
Oh that would make sense. I've had that happen to me before I think. It's annoying that it doesn't auto fix it...